src/constants/channel-rounding.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * List of rounding behaviors when opening multiple channels and encountering a channel that
+ * would be uneconomically small.
+ *
+ * @constant
+ * @type {Object<key, String>}
```

Comment:
lowercase `string`. unfortunately jsdoc doesnt do a good job for complex types like this